openhtf/util/argv.py

```diff
@@ -19,5 +35,7 @@ def __init__(self, *args, **kwargs):
     def __call__(self, parser, namespace, values, option_string=None):
         if hasattr(self, '_proxy'):
             values = self._proxy(parser, namespace, values)
-        setattr(__import__(self._tgt_mod), self._tgt_attr, values)
+        base, mod = self._tgt_mod.rsplit('.', 1)
```

Comment:
hm, yeah it's nonsensical to have no dots, but something one might accidentally do if you forget to do the %s/__name__ business, maybe throw in a quick format check on the inputs of `add_argument()` to fail-fast rather than raise an obscure error here?